vendor/export-test-results.js

```diff
@@ -4,4 +4,7 @@ window.global_test_results = null;
 var exportTestResultsForSauce = function(testResults) {
   window.global_test_results = testResults;
 }
-QUnit.done(exportTestResultsForSauce);
+//Prevent those not using QUnit from having Broccoli imports broken
+if (typeof QUnit != 'undefined') {
+	QUnit.done(exportTestResultsForSauce);
+};
```

Comment:
This not passing jshint, valid would be:
```
if (typeof QUnit !== 'undefined') {
	QUnit.done(exportTestResultsForSauce);
}
```

Could you also remove the comment? I think the intention is quite clear.